contents/blog/the-posthog-array-1-42-0 copy.md

```diff
@@ -0,0 +1,79 @@
+---
+date: 2023-01-16
+title: "Array 1.43.0: HIGHLIGHTS"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ['Release notes', 'Product updates']
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+---
+
+Want to know more about what we're up to? Check out [our roadmap](/roadmap) to see what we're working on and what new beta features are available! You can also subscribe to [our Hogmail newsletter](/newsletter)!
+
+## Update guide for self-hosted users
+
+Any notes about async migrations, etc. go here.
+
+## PostHog 1.43.0 release notes
+
+**Release highlights:**
+
+- [New: Item 1](#new-item-1)
+- [New: Item 2](#new-item-2)
+- [New: Item 3](#new-item-3)
+- [Improved: Item 4](#improved-item-4)
+- [Apps: Item 3](#apps-item-5)
+
+### New: Item 1
+
+@Karl to add a sentence or two here. 
+
+### New: Item 2
+
+@Karl to add a sentence or two here. 
+
+### New: Item 3
+
+@Karl to add a sentence or two here. 
+
+### Improved: Item 4
```

Comment:
```suggestion
### Improved: Experiment creation flow

Ever tried creating an experiment, and realised you made an implementation mistake, or chose the wrong metrics to track?

We've incorporated human errors into the workflow now. Now, you can:

1. Restart a running experiment from now onwards, while keeping everything the same and using the same feature flag key.
2. Change the experiment main metric on a running experiment.
3. Change the secondary metrics on a running experiment.

and finally, as a bonus, you can now have up to 9 variants in an experiment, instead of the usual 3.
```